packages/nocodb-sdk/src/lib/columnHelper/columns/LongText.ts

```
@@ -0,0 +1,43 @@
+import AbstractColumnHelper, {
+  SerializerOrParserFnProps,
+} from '../column.interface';
+
+export class LongTextHelper extends AbstractColumnHelper {
+  columnDefaultMeta = {};
+
+  public serializeValue(value: any) {
+    // This is to remove the quotes added from LongText
+    // TODO (refactor): remove this when we have a better way to handle this
+    value = value?.toString() ?? null;
+    if (value && value.match(/^".*"$/)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)